### PR TITLE
Store Wheel signal and Visual and Airpuff stimuli as TimeSeries

### PR DESCRIPTION
**Pull request by alessandratrapani**
Change Benisty2022Spike2**Events**Interface in Benisty2022Spike2**Signals**Interface to include TimeSeries for:
- VisualStimulus
- AirpuffStimulus
- WheelMotion

### CODE REVIEW

File: src/higley_lab_to_nwb/benisty_2022/benisty_2022_spike2events_interface.py [hunk 1]
```diff
@@ -75,34 +87,46 @@ def get_metadata(self) -> dict:
...
+
+        for stream_id, stream_name in self.behavioral_stream_ids_to_names_map.items():
+            extractor = CedRecordingExtractor(file_path=str(self.source_data["file_path"]), stream_id=stream_id)
+            gain, offset = _get_stream_gain_offset(file_path=str(self.source_data["file_path"]), stream_id=stream_id)
+            behavioral_time_series = TimeSeries(
+                name=stream_name,
+                data=extractor.get_traces(),
+                rate=extractor.get_sampling_frequency(),
+                description=f"The {stream_name} measured over time.",
+                unit="volts",
```
Comment (weiglszonja) [inline src/higley_lab_to_nwb/benisty_2022/benisty_2022_spike2events_interface.py:127]: https://nwbinspector.readthedocs.io/en/dev/best_practices/time_series.html?highlight=ElectricalSeries#units-of-measurement
```suggestion
                unit="Volts",
```

**Comment by weiglszonja** [comment]
Looks good, let me know which file to check.

**Comment by alessandratrapani** [comment]
> Looks good, let me know which file to check.

Sorry [here](https://catalystneuro-processing.s3.us-east-2.amazonaws.com/higley-lab-to-nwb/11222019_grabAM05_spont.nwb)

**Comment by weiglszonja** [comment]
How big would be a fully converted file? if too big to upload, can you show how VisualStimulus and AirpuffStimulus looks like for the whole or longer period of recording?

**Comment by alessandratrapani** [comment]
> How big would be a fully converted file? if too big to upload, can you show how VisualStimulus and AirpuffStimulus looks like for the whole or longer period of recording?
> 
> I remember you also had a figure, but I can't find it now 😅

Yeah, I just opened an issue to investigate these two signals better. Actually, I think there is no Airpuff stimulus (this is a question I will ask them), but I created the interface in a way that is flexible with respect of the signals it can convert. 

[Here](https://github.com/catalystneuro/higley-lab-to-nwb/blob/main/src/higley_lab_to_nwb/benisty_2022/notes/behavioral_signals.png) is the image.

**Comment by weiglszonja** [comment]
Ah yeah, looks like there is no air puff for this session. I would ask for a session where there is air puff stimulus, also another of the visual stimulus would be better. I think the reason #10 this is not working is because the rising frame is too small ? Also why these two are not added by storing the onset times in an EventsTable?

**Comment by alessandratrapani** [comment]
> Ah yeah, looks like there is no air puff for this session. I would ask for a session where there is air puff stimulus, also another of the visual stimulus would be better. I think the reason #10 this is not working is because the rising frame is too small ? Also why these two are not added by storing the onset times in an EventsTable?

I couldn't see why get_rising_frames_from_ttl() was not working with these two signals. I found the reason and it's pretty simple: I was converting a session where no visual stimulus was delivered but the channel was still recording. I uploaded here a stub for the [session without visual stim](https://catalystneuro-processing.s3.us-east-2.amazonaws.com/higley-lab-to-nwb/11222019_grabAM05_spont.nwb) and the [session with visual stim](https://catalystneuro-processing.s3.us-east-2.amazonaws.com/higley-lab-to-nwb/11222019_grabAM05_vis_stim.nwb)

**Review by weiglszonja — APPROVED** [review]
LGTM!

**Pull request merged**